src/routes/userRoutes.js

```diff
@@ -77,9 +78,32 @@ routes.deleteUser = async (req, res) => {
       logger.info('Succeded deleting user account', req.params.id)
       return res.status(200).json('Succeded deleting user account')
```

Comment:
Could you send an object for your 200?

```
return res.status(200).json({success: {message: ''Succeded deleting user account}})

Reply:
In my opinion, this object is not necessary. Since success is success, do need to repeat what you succeed, it’s all in the description of the test. Well errors were generated for many reasons, that’s why we need to specify in the error message.